SurfaceTopography/Generic/ScaleDependentStatistics.py

```diff
@@ -121,19 +123,29 @@ def scale_dependent_statistical_property(self, func, n=1, scale_factor=None, dis
                 iter(scale_factor)
             if distance is not None:
                 iter(distance)
-            return distance, [func(_d) for _d in d]
+            retvals = np.array([func(_d) for _d in d])
         except TypeError:
+            # If there is a single distance, the reliability analysis is skipped
             return distance, func(d)
     else:
         try:
             if scale_factor is not None:
                 iter(scale_factor)
             if distance is not None:
                 iter(distance)
-            return distance, [func(dx, dy) for dx, dy in zip(*d)]
+            retvals = np.array([func(dx, dy) for dx, dy in zip(*d)])
         except TypeError:
+            # If there is a single distance, the reliability analysis is skipped
             return distance, func(*d)
 
+    short_cutoff = self.short_reliability_cutoff() if reliable else None
```

Comment:
similar remark: I think the function short_reliability_cutoff does not clearly state what the distance scale is. To my understanding it is ell = 2 eta dx

Reply:
This is correct, ell = 2 eta dx. I've updated the doc string to explain this.